_layouts/index.html

```diff
@@ -3,6 +3,13 @@
 {% include path/path-assets.html %}
 <head>
 {% include head.html %}
+
+<style type="text/css">
```

Comment:
acest style nu poate sa vina dintr-un partial/ fisier css ca sa nu fie scris aici?

Reply:
ba da, am uitat de el, o sa vad cum pot sa il pun in css direct